### DISCUSSION TITLE
Garden in floors

### DISCUSSION THREAD

**Discussion by LukaszP2** [home-assistant/frontend#20416] [Dashboard]
In 2024.04 was added floors but what about garden?

**Comment by ildar170975** [comment]
People use areas for a garden.
Not ideal since a garden may have sub-areas like "flowerbeds", "entrance area", "lawn", "pond" etc.

**Reply by LukaszP2** [reply]
I know that... I got driveway, terrace and płot. So all of the I want to add to Garden or Outside or Outdoor...

**Reply by Misiu** [reply]
For now, the solution is to add a floor called "Outdoor" and then areas for places. I don't think there will be a new type of place in HA called Garden, we have floors, areas, categories, and labels.
